src/apps/path-finder/store/search-thunk.ts

```diff
@@ -83,6 +84,7 @@ export function searchPath(
         );
 
         toast('Path length is ' + (pathLength + 1));
```

Comment:
The toast would not be needed as we are showing the length on screen

Reply:
Sure
will remove